<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddIndividualWidget</class>
 <widget class="QWidget" name="AddIndividualWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>830</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Species* :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Museum* :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>80</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Hypodigme* :</string>
   </property>
  </widget>
  <widget class="QComboBox" name="speciesComboBox">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>0</y>
     <width>291</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="museumComboBox">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>40</y>
     <width>291</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="hypodigmeComboBox">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>80</y>
     <width>161</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
